lib/game.rb

```diff
@@ -0,0 +1,69 @@
+class Game
+  attr_accessor :board, :player_1, :player_2
+    WIN_COMBINATIONS = [
+      [0, 1, 2],
+      [3, 4, 5],
+      [6, 7, 8],
+      [0, 4, 8],
+      [2, 4, 6],
+      [0, 3, 6],
+      [2, 5, 8],
+      [1, 4, 7]
+
+    ]
+
+  def initialize(player_1=Players::Human.new("X"), player_2=Players::Human.new("O"), board=Board.new)
+    @player_1 = player_1
+    @player_2 = player_2
+    @board = board
+  end
+
+  def current_player
+    board.turn_count % 2 == 0 ? player_1 : player_2
```

Comment:
Consider using [`Integer#even?`](http://ruby-doc.org/core-1.8.7/Integer.html#method-i-even-3F) or [`Integer#odd?`](http://ruby-doc.org/core-1.8.7/Integer.html#method-i-odd-3F) for greater expressiveness.

`@board.turn_count.even? ? @player_1 : @player_2`